src/plugins/liveobjects/livemap.ts

```diff
@@ -40,7 +42,11 @@ export interface LiveMapData extends LiveObjectData {
   data: Map<string, MapEntry>;
 }
 
-export class LiveMap extends LiveObject<LiveMapData> {
+export interface LiveMapUpdate extends LiveObjectUpdate {
+  update: { [keyName: string]: 'updated' | 'deleted' };
```

Comment:
shall we call this update type `removed` instead of `deleted`, to match the internal name we use for the op? (I think we may have discussed this before but I can't remember the outcome)

Reply:
yea, we decided to keep parity with realtime, so `removed`.
this update object in the subscription api DR just had it still as `deleted`, so I used that.
I've updated the DR and will change it here to `removed ` too.